src/providers/wallet/wallet.ts

```diff
@@ -1454,6 +1471,15 @@ export class WalletProvider {
   public clearTxHistory(wallet): void {
     this.invalidateCache(wallet);
     this.persistenceProvider.removeTxHistory(wallet.id);
+    this.clearCache(wallet)
```

Comment:
I'd suggest a more specific name, like `clearHistoryCache` or something. It only clears txs history cache, right?